src/components/userService.ts

```diff
@@ -0,0 +1,42 @@
+import { Memento } from 'vscode';
+
+export class GlobalStateTokenStorage {
+	constructor(private readonly __globalState: Memento) {}
+
+	getUserToken(): string | null {
+		const accessToken = this.__globalState.get('userToken');
```

Comment:
for clarity, I'd call it `this.__globalState.get('accessToken');`

Reply:
addressed